### PR TITLE
Include in Parser a Review Manager

### LINKED ISSUE #1: Parse "Review Manager" from proposals
We currently don't parse the "review manager" field.

### PR DESCRIPTION
**Pull request by digoreis**
I include Review Manager in Parser and Proposal file. I prepared to recipe multiple Reviewers like Authors. 

Also, include a reviewBy in the proposal to help filter. 

Closes #1

### CODE REVIEW

**Review by unnamedd — APPROVED** [review]

File: Sources/Parser.swift [hunk 1]
```diff
@@ -83,14 +88,26 @@ func proposalFrom(fileContents: String, fileName: String) -> Proposal {
 
     let status = statusFromLine(statusLine)
     let words = wordCount(fromFile: fileContents)
-
+    
+    let reviewManagers = reviewManagerFromString(reviewManagerLine)
+    dump(Proposal(title: title,
```
Comment (jessesquires) [inline Sources/Parser.swift]: nit: remove?

File: Sources/Parser.swift [hunk 2]
```diff
@@ -83,14 +88,26 @@ func proposalFrom(fileContents: String, fileName: String) -> Proposal {
...
     return Proposal(title: title,
                     seNumber: seNumber,
                     authors: authors,
+                    reviewManagers: reviewManagers,
                     status: status,
                     fileName: fileName,
                     fileContents: fileContents,
-                    wordCount: words)
+                    wordCount: words
+                    )
```
Comment (jessesquires) [inline Sources/Parser.swift]: let's put `)` on the same line to follow existing project style 😄

File: Sources/Proposal.swift [hunk 3]
```diff
@@ -88,6 +95,17 @@ extension Proposal {
 }
 
 extension Proposal {
+    public func reviewBy(_ reviewer: ReviewManager) -> Bool {
+        return reviewManagers.contains(reviewer)
+    }
+    
+    public func reviewBy(_ reviewerName: String) -> Bool {
```
Comment (jessesquires) [inline Sources/Proposal.swift]: let's make these `reviewedBy` 😄

File: Sources/ReviewManager.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+//
+//
+//  GitHub
+//  https://github.com/jessesquires/swift-proposal-analyzer
+//
+//
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
```
Comment (jessesquires) [inline Sources/ReviewManager.swift:13]: let's copy the project comment headers

**Review by jessesquires — CHANGES_REQUESTED** [review]
Thanks so much @digoreis ! 🎉 

Overall this looks great, just a few minor things to address.

Also -- could we revert the proposal/submodule changes? That will make this cleaner for git history.

**Comment by digoreis** [comment]
Hi @jessesquires, I updated the pull request. To fix the proposal in git history I remove of the project and include de directory in the gitignore file. But the better solution is to remove a copy in the update, creating a symbolic link to the swift-proposal directory. 

And I fixed the others comments in Code Review. Sorry the dump, I forget to remove the dump 😢.

File: .gitignore [hunk 5]
```diff
@@ -64,3 +64,4 @@ fastlane/screenshots
 # https://github.com/johnno1962/injectionforxcode
 
 iOSInjectionProject/
+swift-proposal-analyzer.playground/Resources/proposals/
```
Comment (jessesquires) [inline .gitignore:67]: we don't want to do this, or delete the proposals... 😅

**Comment by jessesquires** [comment]
Thanks @digoreis !

Looks like more unrelated changes were added. We definitely don't want to change the `gitignore`.

Do you want to re-pull the repo, and do a fresh PR with only the changes to the framework source files? 

Just *don't* run the `update_proposals.sh` script and it should be ok.

**Comment by jessesquires** [comment]
Hey @digoreis - did you want open up a new PR for this? 😄

**Pull request closed without merging**